django/thunderstore/social/api/experimental/views/overwolf.py

```diff
@@ -82,6 +83,31 @@ def post(self, request: Request) -> HttpResponse:
         )
 
 
+class OwLogoutRequestBody(serializers.Serializer):
+    sessionid = serializers.CharField(label="Thunderstore session key")
```

Comment:
Why are we not relying on the value from the headers, as is parsed by `UserSessionTokenAuthentication`? Seems that'd be more appropriate here.

Reply:
For one thing, authorization header was already taken by `OauthSharedSecretPermission`. But since it's no longer used, neither was relaying the value in body.